### PR TITLE
Work around the case where an Enttec widget doesn't respond.

### LINKED ISSUE #341: Occasional hangs when doing full discovery via the ENTTEC USBDMX Pro
Hi,

I get occasional hangs when doing full discovery via the ENTTEC USBDMX Pro. The log shows the following:

user.info: Feb 19 16:00:38 olad: EnttecUsbProWidget.cpp:280: Full discovery triggered
user.info: Feb 19 16:00:38 olad: EnttecUsbProWidget.cpp:324: Un-muting all devices
user.info: Feb 19 16:00:38 olad: EnttecUsbProWidget.cpp:343: Sending DUB packet: 0000:00000000 - ffff:ffffffff

After that, it hangs forever. I see several "break too short" messages using the logic sniffer. I need to restart the OLA daemon in order to use OLA again. The issue _might_ be caused by noise on the line or a timing issue with one of my RDM responders, however, OLA should handle this gracefully and not simply hang forever.

### PR DESCRIPTION
**Pull request by nomis52**
This should close #341

### CODE REVIEW

File: plugins/usbpro/EnttecUsbProWidget.cpp [hunk 1]
```diff
@@ -828,6 +847,7 @@ EnttecPort *EnttecUsbProWidgetImpl::GetPort(unsigned int i) {
  */
 bool EnttecUsbProWidgetImpl::SendCommand(uint8_t label, const uint8_t *data,
                                          unsigned int length) {
+  OLA_DEBUG << "TX: " << IntToString(label) << ", length " << length;
```
Comment (peternewman) [inline plugins/usbpro/EnttecUsbProWidget.cpp]: Does this want to stay?

File: plugins/usbpro/EnttecUsbProWidget.cpp [hunk 2]
```diff
@@ -838,8 +858,7 @@ bool EnttecUsbProWidgetImpl::SendCommand(uint8_t label, const uint8_t *data,
 void EnttecUsbProWidgetImpl::HandleMessage(uint8_t label,
                                            const uint8_t *data,
                                            unsigned int length) {
-  OLA_DEBUG << "Enttec received label " << static_cast<uint32_t>(label)
-            << ", length " << length;
+  OLA_DEBUG << "RX: " << IntToString(label) << ", length " << length;
```
Comment (peternewman) [inline plugins/usbpro/EnttecUsbProWidget.cpp]: Likewise?

**Comment by peternewman** [comment]
0.9 and NEWS?

Generally LGTM

File: plugins/usbpro/EnttecUsbProWidget.cpp (hunk 1 above)
Comment (nomis52) [inline plugins/usbpro/EnttecUsbProWidget.cpp]: Yes, I removed the ones from the port above and added these.

File: plugins/usbpro/EnttecUsbProWidget.cpp (hunk 2 above)
Comment (nomis52) [inline plugins/usbpro/EnttecUsbProWidget.cpp]: Yup

**Comment by peternewman** [comment]
What about cherry-picking into 0.9 @nomis52 ?

**Comment by nomis52** [comment]
I think we should instead start to focus on getting 0.10 out. There have been enough changes.

**Pull request merged**